libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2521,9 +2521,7 @@ void IRGeneratorForStatements::appendBareCall(
 		</needsEncoding>
 
 		let <success> := <call>(<gas>, <address>, <?+value> <value>, </+value> <pos>, <length>, 0, 0)
-		<?+returndataVar>
-			let <returndataVar> := <extractReturndataFunction>()
-		</+returndataVar>
+		let <returndataVar> := <extractReturndataFunction>()
```

Comment:
This check is now not done by the caller. Check is always done in callee i.e., extractReturndataFunction